test/unit/test-standard-actions.js

```diff
@@ -395,12 +395,8 @@ describes.sandboxed('StandardActions', {}, () => {
   });
 
   describe('"toggleClass" action', () => {
-    const dummyClass = 'test-class-toggle';
-    const dummyInternalClasses = [
-      'i-amphtml-test-class-toggle',
-      '-amp-test-class-toggle',
-      'amp-test-class-toggle',
-    ];
+    const dummyClass = 'amp-test-class-toggle';
```

Comment:
Can you actually add `test-class-toggle` and `something-amp-class` to this test so we verify that these classes are indeed togglable?